reference/system-databases/inspection-summary.md

```diff
@@ -0,0 +1,75 @@
+---
+title: INSPECTION_SUMMARY
+summary: Learn the `INSPECTION_SUMMARY` inspection summary table.
+category: reference
+---
+
+# INSPECTION_SUMMARY
+
+In some scenarios, you might pay attention only to the monitoring summary of specific links or modules. For example, the number of threads for Coprocessor in the thread pool is configured as 8. If the CPU usage of Coprocessor reaches 750%, you can determine that a risk exists that Coprocessor might become a bottleneck in advance. However, some monitoring metrics vary greatly due to different user workloads, so it is difficult to define certain thresholds. It is important to troubleshoot issues in this scenario, so TiDB provides the `inspection_summary` table for link summary.
+
+The structure of the `information_schema.inspection_summary` inspection summary table is as follows:
+
+{{< copyable "sql" >}}
+
+```sql
+desc inspection_summary;
+```
+
+```sql
++--------------+-----------------------+------+------+---------+-------+
+| Field        | Type                  | Null | Key  | Default | Extra |
++--------------+-----------------------+------+------+---------+-------+
+| RULE         | varchar(64)           | YES  |      | NULL    |       |
+| INSTANCE     | varchar(64)           | YES  |      | NULL    |       |
+| METRICS_NAME | varchar(64)           | YES  |      | NULL    |       |
+| LABEL        | varchar(64)           | YES  |      | NULL    |       |
+| QUANTILE     | double unsigned       | YES  |      | NULL    |       |
+| AVG_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
+| MIN_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
+| MAX_VALUE    | double(22,6) unsigned | YES  |      | NULL    |       |
++--------------+-----------------------+------+------+---------+-------+
+```
+
+Field description:
+
+* `RULE`: Summary rules. New rules are being added, and you can execute the `select * from inspection_rules where type='summary'` statement to check the latest rule list.
+* `INSTANCE`: The monitored instance.
+* `METRIC_NAME`: The monitoring metrics name.
```

Comment:
- Please keep the field name consistent with that in the code block.
- Please check the meaning of the this field because it seems inconsistent with the Chinese version `监控表`.
```suggestion
* `METRICS_NAME`: The monitoring metrics name.
```

Reply:
As confirmed with @reafans, `The monitoring metrics name`  is OK.